### PR TITLE
OFAST-184 Learn Page Arrow Buttons Functionality

### PR DESCRIPTION
**Pull request by AsirAAlam**
- learn page arrow buttons now scroll lessons horizontally
- scroll wheel also works

https://github.com/ofast-team/frontend/assets/46213673/eb947dd8-2a59-438a-af49-223a3408944b

### CODE REVIEW

File: src/utils/usePreventBodyScroll.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+const preventDefault = (ev: Event) => {
+  if (ev.preventDefault) {
+    ev.preventDefault()
+  }
+  ev.returnValue = false
+}
+
+const enableBodyScroll = () => {
+  document && document.removeEventListener('wheel', preventDefault, false)
```
Comment (gblake31) [inline src/utils/usePreventBodyScroll.tsx:11]: woah, cool!

**Comment by gblake31** [comment]
I'm assuming those extra buttons aren't in the code anymore. (I don't see them anywhere)

File: src/pages/LearnPage.tsx [hunk 2]
```diff
@@ -47,16 +57,96 @@ const NextButton = styled(IconButton)({
   },
 })
 
+type scrollVisibilityApiType = React.ContextType<typeof VisibilityContext>
+
+function onWheel(apiObj: scrollVisibilityApiType, ev: React.WheelEvent): void {
+  const isThouchpad = Math.abs(ev.deltaX) !== 0 || Math.abs(ev.deltaY) < 15
```
Comment (gblake31) [inline src/pages/LearnPage.tsx:63]: Looks like a misspelling here.

File: src/pages/LearnPage.tsx [hunk 3]
```diff
@@ -47,16 +57,96 @@ const NextButton = styled(IconButton)({
...
+        >
+          {lesson.name}
+        </Link>
+      </LessonButton>
+    </>
+  )
+}
+
+function LeftArrow() {
+  const visibility = React.useContext<publicApiType>(VisibilityContext)
```
Comment (gblake31) [inline src/pages/LearnPage.tsx:100]: I don't understand what 'publicApiType' does, but I trust you. Seems like a placeholder.

File: src/utils/hideScrollbar.css [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.react-horizontal-scrolling-menu--scroll-container::-webkit-scrollbar {
+  display: none;
+}
+
+.react-horizontal-scrolling-menu--scroll-container {
+  -ms-overflow-style: none; /* IE and Edge */
+  scrollbar-width: none; /* Firefox */
```
Comment (gblake31) [inline src/utils/hideScrollbar.css:7]: Ooh! I wonder if this would fix the landing page issue with firefox.

**Review by gblake31 — APPROVED** [review]

File: src/pages/LearnPage.tsx (hunk 2 above)
Comment (AsirAAlam) [reply src/pages/LearnPage.tsx:63]: Yeah. This is copied code. Fixing it.

File: src/pages/LearnPage.tsx (hunk 3 above)
Comment (AsirAAlam) [reply src/pages/LearnPage.tsx:100]: Copied code. But it's the type of the visibility context, which tells us if the button should be disabled.

**Pull request merged**